src/state/saga/auth.saga.jsx

```diff
@@ -0,0 +1,46 @@
+import axios from "axios";
+import {all, call, takeLatest} from '@redux-saga/core/effects';
+
+import {setAuthTokenCookie} from "../../tool/auth.util";
+import {getAuthorizeRequest} from "../tool/api/auth.endpoint";
+import {RESPONSE_STATUS} from "../../tool/constant";
+import {GET_AUTH_TOKEN} from "../action/auth.actions";
+
+export function* performAuthToken({username, password, navigate, onError, onComplete}) {
+
+    try {
+        const [endpoint, requestOptions] = getAuthorizeRequest({username, password});
```

Comment:
You're destructuring the username and password within the action and then creating an object with that data again here, perhaps you should rather construct the object in the action and then just pass it through here?